sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -20,152 +21,208 @@ public ConversationAnalysisClientLiveTests(bool isAsync, ConversationsClientOpti
         [RecordedTest]
         public async Task AnalyzeConversation()
         {
-            ConversationalTask conversationalTask = new(
-                new ConversationAnalysisOptions(new TextConversationItem("1", "1", "Send an email to Carol about the tomorrow's demo")),
-                new ConversationTaskParameters(TestEnvironment.ProjectName, TestEnvironment.DeploymentName));
+            var data = new
+            {
+                analysisInput = new
+                {
+                    conversationItem = new
+                    {
+                        text = "Send an email to Carol about the tomorrow's demo",
+                        id = "1",
+                        participantId = "1",
+                    }
+                },
+                parameters = new
+                {
+                    projectNam = TestEnvironment.ProjectName,
+                    deploymentName = TestEnvironment.DeploymentName,
+                },
+                kind = "Conversation",
+            };
 
-            Response response = await Client.AnalyzeConversationAsync(conversationalTask.AsRequestContent());
+            Response response = await Client.AnalyzeConversationAsync(RequestContent.Create(data));
 
             // assert - main object
             Assert.IsNotNull(response);
 
             // deserialize
-            using JsonDocument json = await JsonDocument.ParseAsync(response.ContentStream);
-            ConversationalTaskResult conversationalTaskResult = ConversationalTaskResult.DeserializeConversationalTaskResult(json.RootElement);
+            dynamic conversationalTaskResult = response.Content.ToDynamic(DynamicJsonNameMapping.PascalCaseGetters);
             Assert.IsNotNull(conversationalTaskResult);
 
             // assert - prediction type
-            Assert.AreEqual(ProjectKind.Conversation, conversationalTaskResult.Result.Prediction.ProjectKind);
+            Assert.AreEqual("Conversation", conversationalTaskResult.Result.Prediction.ProjectKind?.ToString());
```

Comment:
Would you consider using a cast to string here?  Since you're comparing it to a string, it seems like the cast shouldn't fail.